worker/src/response.rs

```diff
@@ -472,3 +472,35 @@ impl From<web_sys::Response> for Response {
         }
     }
 }
+
+/// A trait used to represent any viable Response struct that can be used in the Worker.
+/// The only requirement is that it be convertable to a web_sys::Response.
+pub trait WorkerResponse {
```

Comment:
Should we adopt a more generic trait name like `IntoResponse::into_response` from `axum`? I think ideally this would just be `From<_>`, but I supposed that cannot be implemented for external types.

Reply:
done in https://github.com/cloudflare/workers-rs/pull/530/commits/c08eaa344c74e7e9d7076e597611263cb020eeda

I tried a few different ways to just use a regular trait bound, so it would work for any From/Into web_sys, but kept running into the orphan rule or a restriction that would force consumers to impl something else anyway. I wouldn't be surprised if someone finds an elegant way around that, but I didn't see it for now 🤷